lib/public/views/lhcPeriods/ActiveColumns/lhcPeriodsActiveColumns.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { h } from '/js/src/index.js';
+
+/**
+ * List of active columns for a generic periods table
+ */
+export const lhcPeriodsActiveColumns = {
+    name: {
+        name: 'Name',
+        visible: true,
+        format: (_, lhcPeriod) => [h('td.text-ellipsis', lhcPeriod.name)],
```

Comment:
Please try to have a look on other tables how it work. You will find out that some tables already have elipsis, and without having to put `td` in `td` 😃

Reply:
removed